src/os/bluestore/BitMapAllocator.cc

```diff
@@ -24,6 +24,7 @@ BitMapAllocator::BitMapAllocator(int64_t device_size, int64_t block_size)
     m_num_committing(0)
 {
   int64_t zone_size_blks = g_conf->bluestore_bitmapallocator_blocks_per_zone;
+  assert((zone_size_blks & (zone_size_blks - 1)) == 0);
```

Comment:
Can you log a fatal message as well and return? assert might be void in case of optimized builds.